### DISCUSSION TITLE
A Good Start

### DISCUSSION THREAD

**Discussion by MuditJainRepos** [NVIDIA/open-gpu-kernel-modules#158] [General]
Congratulations People from Nvidia and Members of the Community Coming Here !!!
I Hope we can make something good out of this Effort.
I am using this as a starting point for meaningful discussions that can take place here and be referenced in Future.

**Comment by master7720** [comment]
when i use the bathroom i fall in the toilet could this be an issue?

**Comment by sTiKyt** [comment]
> when i use the bathroom i fall in the toilet could this be an issue?

Only when you mention it in software related discussions
